classes/03class/exercises/c03-docker02/mfreitassm/Dockerfile.v2

```diff
@@ -0,0 +1,5 @@
+FROM httpd:2.4.41-alpine
+
+RUN rm /usr/local/apache2/htdocs/*
```

Comment:
You don't need the `RUN` instruction here. It will create an extra layer into your image and the content of this folder will be replaced by the `--volume` instruction anyway.

Reply:
Okay, Thanks @alanlima!